### PR TITLE
Updated the do and dont panels to allow custom titles

### PR DESCRIPTION
**Pull request by pflynny**
Requested by the PIMS and Moderation team – It's a small update allowing custom titles for the do and don't list panels.

### CODE REVIEW

**Comment by AdamChrimes** [comment]
Looks good to me, might be worth @mikemonteith having a quick glance

File: wagtailnhsukfrontend/templates/wagtailnhsukfrontend/do_list.html [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <div class="nhsuk-do-dont-list">
-  <h{{ heading_level }} class="nhsuk-do-dont-list__label">Do</h{{ heading_level }} >
+  <h{{ heading_level }} class="nhsuk-do-dont-list__label">
```
Comment (mikemonteith) [inline wagtailnhsukfrontend/templates/wagtailnhsukfrontend/do_list.html:2]: You can use the "default" tag here instead https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#default

` {{ label|default:"Do" }}`

File: wagtailnhsukfrontend/blocks.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ class Meta:
 class DontBlock(FlattenValueContext, StructBlock):
 
     heading_level = IntegerBlock(required=True, min_value=2, max_value=4, default=3, help_text='The heading level affects users with screen readers. Default=3, Min=2, Max=4.')
-
+    label = CharBlock(label='Heading', required=False, help_text='Adding a label here will overwrite the default of Dont')
```
Comment (mikemonteith) [inline wagtailnhsukfrontend/blocks.py:103]: Dont should have an apostrophe. `Don't`

File: wagtailnhsukfrontend/templates/wagtailnhsukfrontend/dont_list.html [hunk 3]
```diff
@@ -1,5 +1,11 @@
 <div class="nhsuk-do-dont-list">
-  <h{{ heading_level }} class="nhsuk-do-dont-list__label">Don&#39;t</h{{ heading_level }}>
```
Comment (mikemonteith) [inline wagtailnhsukfrontend/templates/wagtailnhsukfrontend/dont_list.html:2]: I'm not sure why we're using `&#39;` here. Should be using the unicode "RIGHT SINGLE QUOTATION MARK" (’) https://www.fileformat.info/info/unicode/char/2019/index.htm

File: wagtailnhsukfrontend/templates/wagtailnhsukfrontend/do_list.html (hunk 1 above)
Comment (pflynny) [reply wagtailnhsukfrontend/templates/wagtailnhsukfrontend/do_list.html:2]: Nice one, I didn't think of that.

File: wagtailnhsukfrontend/blocks.py (hunk 2 above)
Comment (pflynny) [reply wagtailnhsukfrontend/blocks.py]: Good spot!

File: wagtailnhsukfrontend/templates/wagtailnhsukfrontend/dont_list.html (hunk 3 above)
Comment (pflynny) [reply wagtailnhsukfrontend/templates/wagtailnhsukfrontend/dont_list.html:2]: Yeah, that is ugly – I've fixed that.

**Pull request merged**